ui/app/scripts/stream/controllers/stream.js

```diff
@@ -23,6 +23,12 @@ define(function (require) {
     'use strict';
 
     var joint = require('joint');
+    var angular = require('angular');
```

Comment:
Not sure that I like so much code in a Controller ... @BoykoAlex Should probably be refactored into its own service. Maybe a follow-up issue-ticket?